tests/vmi_lifecycle_test.go

```diff
@@ -1307,7 +1307,7 @@ var _ = Describe("[rfe_id:273][crit:high][arm64][vendor:cnv-qe@redhat.com][level
 
 				By("Creating a VirtualMachineInstance with different namespace")
 				vmi = libvmi.New(
-					libvmi.WithResourceMemory("256Mi"),
+					libvmi.WithResourceMemory("1Mi"),
```

Comment:
I am not sure about this one, but per what I remember, on the ARM architecture, it failed to load (even before boot) with a low memory. I think we just need confirmation about it.
Lets trigger the ARM job and see (this whole test group is marked to run on the ARM job).

Reply:
Your execution passed https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/7921/pull-kubevirt-e2e-arm64/1536944112983347200#1:build-log.txt%3A1557 but I am not sure if this test is running there at all.

Reply:
I was under the impression that `arm64` is marked at the top level `Describe`. So it should run.

Reply:
So would you approve this PR? It passes on both x86 and arm, and waste a little bit less resources.